app/account/edit/components/account-form.tsx

```diff
@@ -0,0 +1,164 @@
+import { useState } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+import {
+  Box,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  Image,
+  VStack,
+  HStack,
+  Flex,
+  Input
+} from '@chakra-ui/react'
+
+import { MdAccountCircle } from 'react-icons/md'
+import { AccountSchema, accountResolver } from '@/account/schema'
+import { PrimaryButton, SecondaryButton } from '@/components/button'
+import { InputForm } from '@/components/input'
+import { Loading } from '@/components/loading'
+import { useUserId } from '@/providers/session-provider'
+import { useUploadFile, useUserUpdate } from '../../hooks'
+
+export type UserDetailsProps = {
+  name: string
+  email: string
+  profile_picture_url: string | undefined | null
+}
+
+export const AccountEditForm = ({
+  userDetails
+}: {
+  userDetails: UserDetailsProps
+}) => {
+  const userId = useUserId()
+  const { uploadFile } = useUploadFile()
+  const { updateUser, isUserUpdating, isUserUpdatingSuccessfull } =
+    useUserUpdate()
+  const [selectedImage, setSelectedImage] = useState<File | null>(null)
+
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors }
+  } = useForm<AccountSchema>({
+    defaultValues: {
+      name: userDetails?.name,
+      email: userDetails?.email,
+      profile_picture_url: userDetails?.profile_picture_url,
+      uploaded_image_file: null
+    },
+    resolver: accountResolver
+  })
+
+  const createHandler = handleSubmit(async (data: AccountSchema) => {
+    const { name, email } = data
+
+    if (name !== undefined && email !== undefined) {
+      if (selectedImage && userDetails?.profile_picture_url) {
+        await uploadFile(userDetails.profile_picture_url, selectedImage, userId)
+      }
+
+      updateUser(name, email)
+    }
+  })
+
+  return (
+    <>
+      {!isUserUpdatingSuccessfull && !isUserUpdating && (
+        <Box as="form" onSubmit={createHandler}>
+          <Flex direction="column">
+            <VStack align="start" gap="20px">
+              <FormControl isRequired isInvalid={!!errors.email}>
```

Comment:
If you add "isRequired", you can see the red required mark. But it does not show the custom validation error. Could you show the custom validation errors instead of HTML validation error? 

The current validation 
<img width="633" alt="Screenshot 2024-03-02 at 2 35 15 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/e1df121e-b74f-4b89-a095-73cc05f7f222">

Should be like this 
<img width="568" alt="Screenshot 2024-03-02 at 2 36 27 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/5cfde7d5-77d9-4102-9a8e-fcc63f9223c7">